platform/jvm/capture/src/main/kotlin/io/bitdrift/capture/LoggerImpl.kt

```diff
@@ -265,6 +262,36 @@ internal class LoggerImpl(
         appExitSaveCurrentSessionId()
     }
 
+    override fun shutdown() {
+        runCatching {
+            Thread {
+                val shutdownOperations = listOf(
+                    { appExitSaveCurrentSessionId(this.sessionId) },
+                    { eventsListenerTarget.stop() },
+                    { captureExecutors.shutdown() },
+                    { apiClient.shutdown() },
+                    { captureNetwork.shutdown() },
+                    { CaptureJniLibrary.destroyLogger(this.loggerId) },
```

Comment:
🔒 This PR should be locked until we can make this call to rust not fail and crash the app